.github/autosubmit.yml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file will be added to flutter's internal repository.
+# https://github.com/flutter/flutter/wiki/Autosubmit-bot
+config_path: 'autosubmit/flutter/autosubmit_main.yml'
```

Comment:
Should this be autosubmit/engine/autosubmit_main.yml?

Reply:
No. So how it works is that since we use the pointer file we can define a many-to-one relationship with the global files. Really the only one that was different right now is the one for master since flutter has not been moved to that default branch yet. So flutter -> autosubmit/flutter/autosubmit_main.yml and cocoon, engine, packages -> autosubmit/flutter/autosubmit_main.yml since they have the main branch in common.

Reply:
With that being said I could make it clear by putting the files in specific directories within flutter/.github if that is more readable and easier to follow?